bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoBindingConstants.java

```diff
@@ -12,292 +12,124 @@
+    public static final String PROPERTY_CITY = "city";
+    public static final String PROPERTY_COUNTRY = "country";
+    public static final String PROPERTY_TIMEZONE = "timezone";
+    // Channel group ids
+    public static final String GROUP_TEMPERATURE = "temperature";
+    public static final String GROUP_HUMIDITY = "humidity";
+    public static final String GROUP_CO2 = "co2";
+    public static final String GROUP_NOISE = "noise";
+    public static final String GROUP_PRESSURE = "pressure";
+    public static final String GROUP_TIMESTAMP = "timestamp";
+    public static final String GROUP_RAIN = "rain";
+    public static final String GROUP_WIND = "wind";
+    public static final String GROUP_HEALTH = "health";
+    public static final String GROUP_HOME_ENERGY = "energy";
+    public static final String GROUP_SIGNAL = "signal";
+    public static final String GROUP_BATTERY = "battery";
+    public static final String GROUP_HOME_SECURITY = "home-security";
+    public static final String GROUP_WELCOME = "welcome";
+    public static final String GROUP_PRESENCE = "presence";
+    public static final String GROUP_WELCOME_EVENT = "welcome-event";
+    public static final String GROUP_PERSON = "person";
+    public static final String GROUP_PERSON_EVENT = "person-event";
+    public static final String GROUP_ROOM_TEMPERATURE = "room-temperature";
+    public static final String GROUP_ROOM_PROPERTIES = "room-properties";
+    public static final String GROUP_TH_PROPERTIES = "th-properties";
+    public static final String GROUP_TH_SETPOINT = "setpoint";
+
+    // Alternative extended groups
+    public static final String GROUP_EXTENSION = "-extended";
+    public static final String GROUP_EXTENDED_TIMESTAMP = GROUP_TIMESTAMP + GROUP_EXTENSION;
+    public static final String GROUP_BATTERY_EXTENDED = GROUP_BATTERY + GROUP_EXTENSION;
+    public static final String GROUP_PRESSURE_EXTENDED = GROUP_PRESSURE + GROUP_EXTENSION;
+    public static final String GROUP_TEMPERATURE_EXTENDED = GROUP_TEMPERATURE + GROUP_EXTENSION;
+
+    // Channel ids
+    public static final String CHANNEL_VALUE = "value";
+    public static final String CHANNEL_TREND = "trend";
+    public static final String CHANNEL_MAX_TIME = "max-time";
+    public static final String CHANNEL_MIN_TIME = "min-time";
+    public static final String CHANNEL_MAX_VALUE = "max-today";
+    public static final String CHANNEL_MIN_VALUE = "min-today";
+    public static final String CHANNEL_HUMIDEX = "humidex";
+    public static final String CHANNEL_HUMIDEX_SCALE = "humidex-scale";
+    public static final String CHANNEL_DEWPOINT = "dewpoint";
+    public static final String CHANNEL_DEWPOINT_DEP = "dewpoint-depression";
+    public static final String CHANNEL_HEAT_INDEX = "heat-index";
+    public static final String CHANNEL_ABSOLUTE_PRESSURE = "absolute";
+    public static final String CHANNEL_LAST_SEEN = "last-seen";
+    public static final String CHANNEL_MEASURE_TIMESTAMP = "measures";
```

Comment:
Seems at first reading but correct in fact because it belongs to the "timestamp-extended" group so, it's the timestamp of the measures => timestamp#measures by contrast with timestamp#last-seen

Reply:
Ok for the value but in this case, your constant could rather be CHANNEL_MEASURES_TIMESTAMP ?